assets/sass/_responsive-component-table.scss

```diff
@@ -0,0 +1,30 @@
+
+// Components tables
+.has-components-table {
```

Comment:
I'd move this into a new file in sass/pages/_docs.scss, since it's really only used in the docs view.